### ISSUE TITLE
Floating action button can’t show on on iPhone12 mini

### ISSUE THREAD

**Issue by oceanandabyss** [selfire1/obsidian-you-theme#18]
![961D042B-2BEE-4E01-8B4E-F3B0908D5D18](https://user-images.githubusercontent.com/72381254/157142794-c480f1c5-5256-4bde-b46b-68e7aa74e6f3.jpeg)

Just updated the theme yesterday, after that I can’t use the floating action button for now.

**Comment by selfire1** [comment]
Hi there! Does the issue still persist? I am having a hard time recreating it.

**Comment by oceanandabyss** [comment]
Yeah, it’s still there. Nothing change since my last post. Can I offer any info to help? Not sure what did I to my phone version Obsidian to make this issue still there…

![4E4CFF7C-1323-4916-AC28-3D013E93CC81](https://user-images.githubusercontent.com/72381254/164681802-4b41681f-d1ab-463b-be81-f9de5ac140f8.jpeg)
This is from this morning.

**Comment by xbaibs** [comment]
It happens on iPhone Xs too

**Comment by tangyuan233** [comment]
It happens on my iPhone 13pro too.
